build.proj

```diff
@@ -34,8 +34,9 @@
 
     <Stage0Directory>$(RepoRoot)/.dotnet_stage0/$(Architecture)</Stage0Directory>
     <DotnetStage0>$(Stage0Directory)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <ArtifactsDirectory>$(RepoRoot)/artifacts</ArtifactsDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
```

Comment:
As stated above, `PackagesDir` is the name used in Build Tools targets that we import so we cannot change this